pkg/lib/operation/dbt/generate/profile/operation.go

```diff
@@ -33,42 +34,67 @@ func Run(ctx context.Context, targetName string, d dependencies) (err error) {
 	}
 	fs := project.Fs()
 
-	targetUpper := strings.ToUpper(targetName)
-	profileDetails := map[string]interface{}{
-		"target": targetName,
-		"outputs": map[string]interface{}{
-			targetName: map[string]interface{}{
-				"account":   fmt.Sprintf("{{ env_var(\"DBT_KBC_%s_ACCOUNT\") }}", targetUpper),
-				"database":  fmt.Sprintf("{{ env_var(\"DBT_KBC_%s_DATABASE\") }}", targetUpper),
-				"password":  fmt.Sprintf("{{ env_var(\"DBT_KBC_%s_PASSWORD\") }}", targetUpper),
-				"schema":    fmt.Sprintf("{{ env_var(\"DBT_KBC_%s_SCHEMA\") }}", targetUpper),
-				"type":      fmt.Sprintf("{{ env_var(\"DBT_KBC_%s_TYPE\") }}", targetUpper),
-				"user":      fmt.Sprintf("{{ env_var(\"DBT_KBC_%s_USER\") }}", targetUpper),
-				"warehouse": fmt.Sprintf("{{ env_var(\"DBT_KBC_%s_WAREHOUSE\") }}", targetUpper),
-			},
-		},
+	// Load profiles file if exists
+	profilesFile := orderedmap.New()
+	profilesFileDef := filesystem.NewFileDef(dbt.ProfilesPath).SetDescription("dbt profiles")
+	if _, err := fs.FileLoader().ReadYamlFileTo(profilesFileDef, profilesFile); err != nil && !errors.Is(err, filesystem.ErrNotExist) {
+		return err
 	}
-	profilesFile := make(map[string]interface{})
-	profilesFile["send_anonymous_usage_stats"] = false
 
-	if fs.Exists(profilePath) {
-		file, err := fs.ReadFile(filesystem.NewFileDef(profilePath))
-		if err != nil {
-			return err
-		}
-		err = yaml.Unmarshal([]byte(file.Content), &profilesFile)
-		if err != nil {
-			return fmt.Errorf(`profiles file "%s" is not valid yaml: %w`, profilePath, err)
-		}
+	// Set send_anonymous_usage_stats if missing
+	if _, found := profilesFile.Get("send_anonymous_usage_stats"); !found {
+		profilesFile.Set("send_anonymous_usage_stats", false)
 	}
-	profilesFile[project.Profile()] = profileDetails
 
-	yamlEnc, err := yaml.Marshal(&profilesFile)
-	if err != nil {
-		return err
-	}
-	err = fs.WriteFile(filesystem.NewRawFile(profilePath, string(yamlEnc)))
-	if err != nil {
+	// Set profile
+	targetUpper := strings.ToUpper(targetName)
+	profilesFile.Set(project.Profile(), orderedmap.FromPairs([]orderedmap.Pair{
+		{
+			Key:   "target",
+			Value: targetName,
+		},
+		{
+			Key: "outputs",
+			Value: orderedmap.FromPairs([]orderedmap.Pair{
+				{
+					Key: targetName,
```

Comment:
To ako to v tom subore bolo poprehadzovane (podla abecedy) mi hrozne bije do oci, tak som tu pouzil `orderedmap`, (kedze chceme zachovat extra keys).

Kedze som nedavno pridal do orderedmap podporu pre `yaml`, tak uz sa to da.

Manualne otestovane:
![image](https://user-images.githubusercontent.com/19371734/193212557-b2f59bd7-0f89-47fe-ac68-c34fcfe5b189.png)